strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/support/ArtifactRoutingRulesChecker.java

```diff
@@ -0,0 +1,83 @@
+package org.carlspring.strongbox.services.support;
+
+import org.carlspring.strongbox.configuration.Configuration;
+import org.carlspring.strongbox.configuration.ConfigurationManager;
+import org.carlspring.strongbox.storage.routing.RoutingRule;
+import org.carlspring.strongbox.storage.routing.RoutingRules;
+import org.carlspring.strongbox.storage.routing.RuleSet;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Przemyslaw Fusik
+ *
+ * @see <a href="https://github.com/strongbox/strongbox/wiki/Artifact-Routing-Rules">Artifact Routing Rules</a>
+ */
+@Component
+public class ArtifactRoutingRulesChecker
```

Comment:
I should have thought of this before! Thanks! This looks clearer and much neater!